apps/charterafrica/src/components/Hero/Slide.js

```diff
@@ -66,7 +66,7 @@ const Slide = React.forwardRef(function Slide(props, ref) {
             })}
           />
           <LineClampedRichTypography
-            lineClamp={{ xs: "2", sm: "1" }}
+            lineClamp="2"
```

Comment:
Since we're switching to 2 lines everywhere, `sm` lines that limits height to one line needs to be removed as well:
https://github.com/CodeForAfrica/ui/blob/44215e6babef7dcfe1c97efd886b2699469cb11e/apps/charterafrica/src/components/Hero/Slide.js#L77-L79